build.gradle

```diff
@@ -129,7 +129,7 @@ ext {
     androidxWorkVersion = "2.0.1"
 
     daggerVersion = '2.22.1'
-    fluxCVersion = '1.6.24'
+    fluxCVersion = '1.6.24.1'
```

Comment:
Just ot be sure, is this expected that we poin to 1.6.24.1 here while the hotfix in #13205 was pointing to 1.6.22.1, right? I think it is, as I see 1.6.24.1 of FluxC got released since indeed, but I also see that https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1732 was still pending merge, so just want to double-check the git workflow was applied right here.